src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/internal/utils/ItemMetadataFunctions.java

```diff
@@ -60,4 +60,15 @@ public static io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.server
 
         return modelItemMetadata;
     }
+
+    private static String toRFC3339DateTimeString(final Instant time)
+    {
+        if (time.isBefore(Instant.ofEpochSecond(-62167219200L))) {
```

Comment:
In a way this is an improvement as it is not as compute-intensive as parsing a string, but on the other hand this is also harder to understand.  Either way, you're still constructing an `Instant` each time this code is run though, which is every time a date is serialized.  How do we resolve this?

Reply:
I have now created a field instead of constructing this in the method each time and used the more readable parse method